java/runtime/src/main/java/org/ray/runtime/WorkerContext.java

```diff
@@ -48,7 +48,7 @@ public long getNativeWorkerContext() {
    * for other threads, this method returns a random ID.
    */
   public TaskId getCurrentTaskId() {
-    return new TaskId(nativeGetCurrentTaskId(nativeWorkerContextPointer));
+    return TaskId.fromByteBuffer(ByteBuffer.wrap(nativeGetCurrentTaskId(nativeWorkerContextPointer)));
```

Comment:
maybe add a `TaskId.fromBytes` method, so we don't need to use `wrap` everywhere.